src/include/httpfs_curl_client.hpp

```diff
@@ -2,13 +2,13 @@
 
 #include "duckdb/common/http_util.hpp"
 
+#include <curl/curl.h>
```

Comment:
We shouldn't include header files inside of namespace, otherwise all symbols inside are namespace scoped.